var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -196,7 +199,7 @@ def check(self):
     patch("0001-Fix-compilation-error-with-StringRef-to-basic-string.patch", when="@:3.8")
     patch("0002-Fix-rocblas-clients-blas.patch", when="@4.2.0:4.3.1")
     patch("0003-Fix-rocblas-gentest.patch", when="@4.2.0:5.1")
-    patch("0004-Find-python.patch", when="@master:")
+    patch("0004-Find-python.patch", when="@5.2.0:")
```

Comment:
Same comment on patches as above

Reply:
This is specific to 5.2.0 and changes are upstreamed already for 5.3.0 release.

Reply:
Sure, but it is good for package readers to have comments on patches, to have an idea which changes they introduce in the code.

Reply:
Added description in the recipe as below.
The patch 0004-Find-python.patch is to find Python package and set command python as python3.
It has been taken into the 5.2 branch. This patch will no longer needed from 5.3 release.